specs/commandline/version.md

```diff
@@ -0,0 +1,46 @@
+# notation version
+
+## Description
+
+Use `notation version` to print notation version information.
+
+Upon successful execution, the following information is printed.
+
+```text
+Notation: Notary v2, A tool to sign, store, and verify artifacts.
+
+Version:    <version_in_format_vX.Y.Z[suffix]>
+Go version: <version in format goX.Y.Z>
+Git commit: <commit_id_in_7_characters>
```

Comment:
Why do we need to display go version and git commit? Can they be inferred from `Version`?

Reply:
I agree - we should have just the basic version there but if someone wants more, they could perhaps do the -v (verbose) to get additional information.  This is what you get with **kubectl version** for instance:

```
$kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.2", GitCommit:"5835544ca568b757a8ecae5c153f317e5736700e", GitTreeState:"clean", BuildDate:"2022-09-21T14:33:49Z", GoVersion:"go1.19.1", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v4.5.7

$kubectl version --short
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
Client Version: v1.25.2
Kustomize Version: v4.5.7
```